<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoBuyVip</class>
 <widget class="QWidget" name="GoBuyVip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>602</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1162</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>288</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">font: 36pt &quot;宋体&quot;;
color: rgb(255, 211, 1);</string>
           </property>
           <property name="text">
            <string>会员购买</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0,0" columnminimumwidth="1,2,3">
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="1" column="1" colspan="2">
          <widget class="QCheckBox" name="checkBoxSeason">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>季卡，钻石会员，8.5 折/单</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/icon/images/icon/Season.svg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/icon/images/icon/Year.svg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QCheckBox" name="checkBoxYear">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>年卡，皇冠会员，8.0 折/单</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
           </property>
           <property name="text">
            <string>CN￥：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/icon/images/icon/Month.svg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QCheckBox" name="checkBoxMonth">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>月卡，普通会员，9.0 折/单</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="labelMoney">
           <property name="styleSheet">
            <string notr="true">font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(248, 73, 95);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 24pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOk">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 24pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>384</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
